README.md

```diff
@@ -233,10 +233,11 @@ See [Pool Upgrade](docs/pool-upgrade.md) and [Pool Upgrade How To](docs/pool-upg
 - [Deployment Pattern](docs/deployment.png)
 - [Deployment Pattern Detailed](docs/deployment-detailed.png)
 - [Deployment Recommendations](https://github.com/zigbee-alliance/distributed-compliance-ledger/wiki/DCL-MainNet-Deployment)
-- [Running a Node](docs/running-node.md)
-  - [Running a Genesis Validator Node](docs/advanced/running-genesis-node.md)
-  - [Running a Validator Node](docs/advanced/running-validator-node.md)
-  - [Running an Observer Node](docs/advanced/running-observer-node.md)
+- [Running Node in a new network](docs/running-node.md)
+  - [Running Genesis Validator Node](docs/advanced/running-genesis-node.md)
+  - [Running Validator Node](docs/advanced/running-validator-node.md)
+  - [Running Observer Node](docs/advanced/running-observer-node.md)
+- [Running Node in an existing network](docs/running-node-in-existing-network.md)
```

Comment:
Please also look at lines 203-205 in this files. I believe it also requires some adjustment.